conf/test.config

```diff
@@ -34,7 +34,6 @@ params {
 
     // Observations
     observations_id_col        = 'sample'
-    observations_name_col      = 'sample'
```

Comment:
Would rather you left this as an example, even if your config changes make it not necessary.

Reply:
Can do, but then I'll have the example commented out, otherwise the param will continue to have a default which might not work for some input data